charts/generic-service/README.md

```diff
@@ -157,7 +157,8 @@ app:
 | `alerting.http.referenceInterval`               | `1w`                        | The time interval to to compare with the sample interval to detect changes                               |
 | `alerting.http.maxSlowdown`                     | `2.5`                       | The maximum HTTP response slowdown in the sample interval compared to the reference interval             |
 | `alerting.http.max4xxRatio`                     | `2.5`                       | The maximum HTTP 4xx ratio increase in the sample interval compared to the reference interval            |
-| `alerting.http.max5xxCount`                     | `0`                         | The maximum number of HTTP 5xx responses in the sample interval                                          |
+| `alerting.http.max5xxCount`                     | `0`                         | The maximum number of HTTP 5xx responses (except 504) in the sample interval                             |
+| `alerting.http.maxTimeoutCount`                 | `0`                         | The maximum number of HTTP gateway timeout responses (504) in the sample interval                        |
```

Comment:
Please add this in `values.schema.json` too.

Reply:
Good catch, done.